container/container_unix.go

```diff
@@ -485,16 +486,18 @@ func (container *Container) CopyImagePathContent(v volume.Volume, destination st
 		return err
 	}
 
-	path, err := v.Mount()
+	id := stringid.GenerateNonCryptoID()
```

Comment:
It'd be pretty catastrophic for me (the bug that got us here in the 
first place) if there were duplicates. Can we just ensure no other 
containers have the ID and re-roll if needed?

-Erik

On 8 Mar 2016, at 12:25, Brian Goff wrote:

>> @@ -485,16 +486,18 @@ func (container *Container) 
>> CopyImagePathContent(v volume.Volume, destination st
>> 		return err
>> 	}
>>
>> -	path, err := v.Mount()
>> +	id := stringid.GenerateNonCryptoID()
>
> We generally use non-crypto for these sort of IDs (even container ID's 
> are non-crypto). These are going to be relatively short-lived and not 
> likely to conflict (and not catastrophic if they happen to).
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/docker/docker/pull/21015/files#r55421645